ep/build-velox/src/build_velox.sh

```diff
@@ -120,11 +120,18 @@ function compile {
     COMPILE_OPTION="$COMPILE_OPTION -DVELOX_ENABLE_GCS=ON"
   fi
 
+  if [[ $ENABLE_TESTS == "ON" ]] || [[ $ENABLE_BENCHMARK == "ON" ]]; then
```

Comment:
Is this part necessary? I note `VELOX_ENABLE_PARSE=ON` if `VELOX_BUILD_TESTING` or `VELXO_BUILD_BENCHMARKS`  is ON according to CMakeLists.txt in velox. Maybe, we can try to remove this part to see CI feedback. Thanks!

Reply:
@PHILO-HE This is necessary because in Velox, VELOX_ENABLE_PARSE is set to ON by default, while in Gluten, VELOX_ENABLE_DUCKDB is set to OFF by default. Therefore, if tests are not enabled, Velox compilation will exclude DuckDB but include Parse, resulting in a compilation issue where DuckDB files cannot be found.